spec/models/need_spec.rb

```diff
@@ -17,11 +17,11 @@
     let!(:uncompleted_need) { create :need }
     let!(:completed_need) { create :need, :completed }
 
-    it '.completed' do
-      expect(described_class.uncompleted).to eq [uncompleted_need]
+    it '.uncompleted' do
```

Comment:
I'm not sure how this is working in the pipeline or on others peoples machines - this broke for me so applying a fix.  If this is not correct please let me know and I'll revert.